lib/node_modules/@stdlib/math/base/special/cexp/README.md

```diff
@@ -99,16 +101,14 @@ var re;
 var im;
```

Comment:
Also, we can remove the `real` and `imag` imports, as they are no longer used in this example.